UnicornEngine/video/include/unicorn/video/Camera.hpp

```diff
@@ -44,61 +45,73 @@ class Camera
      * @param zfar Far frustrum plane
      */
     UNICORN_EXPORT void SetPerspective(float fov, float aspect, float znear, float zfar);
+
     /**
      * @brief Sets aspect ratio
      * @param aspect Aspect value
      */
     UNICORN_EXPORT void SetAspectRatio(float aspect);
+
     /**
      * @brief Changes camera position by given vector
      * @param delta Vector of translation
      */
-    UNICORN_EXPORT void Translate(const glm::vec3& delta);
+    UNICORN_EXPORT void Translate(glm::vec3 const& delta);
+
     /**
      * @brief Sets direction
      * @param direction New direction value
      */
-    UNICORN_EXPORT void SetDirection(const glm::vec3& direction);
+    UNICORN_EXPORT void SetDirection(glm::vec3 const& direction);
+
     /**
      * @brief Sets up vector
      * @param upVector New up vector value
      */
-    UNICORN_EXPORT void SetUpVector(const glm::vec3& upVector);
+    UNICORN_EXPORT void SetUpVector(glm::vec3 const& upVector);
+
     /**
      * @brief Sets position
      * @param position New position value
      */
-    UNICORN_EXPORT void SetPosition(const glm::vec3& position);
+    UNICORN_EXPORT void SetPosition(glm::vec3 const& position);
```

Comment:
I am pretty sure all the trivial member fields could be public

Reply:
No, look at .cpp

Reply:
Okay.